definitions/contested/json/CaseTypeTab/CaseTypeTab-respSol-nonprod.json

```diff
@@ -0,0 +1,182 @@
+[
+  {
+    "LiveFrom": "01/02/2021",
+    "CaseTypeID": "FinancialRemedyContested",
+    "TabID": "Trial Bundle",
+    "TabLabel": "Trial Bundle",
+    "TabDisplayOrder": 20,
+    "CaseFieldID": "appHearingBundlesCollection",
+    "TabFieldDisplayOrder": 1
+  },
+  {
+    "LiveFrom": "01/02/2021",
+    "CaseTypeID": "FinancialRemedyContested",
+    "TabID": "Applicant Documents",
+    "TabLabel": "Applicant Documents",
+    "TabDisplayOrder": 18,
+    "CaseFieldID": "appFormEExhibitsCollection",
+    "TabFieldDisplayOrder": 2
```

Comment:
will these appear above the old collections in the applicant documents tab? if so, do you not need to update the display order for the old collections?

Reply:
Good spot. Updated old collections order!